src/i_oplmusic.c

```diff
@@ -1413,7 +1415,19 @@ static void TrackTimerCallback(void *arg)
 
         if (running_tracks <= 0 && song_looping)
         {
-            OPL_SetCallback(5000, RestartSong, NULL);
```

Comment:
Why ist this?

Reply:
This was the hackiest part of my PR. `song_looping` is supposed to be a boolean, but I'm now interpreting values above 2 to mean (loop a finite number of times). The actual loop count is `song_looping` - 2. Values of 0 or 1 keep the original behavior, don't loop or loop forever.

I should really put the loop count in a separate variable.